scipy/linalg/flapack_gen_tri.pyf.src

```diff
@@ -66,6 +66,32 @@ subroutine <prefix>gttrs(trans, n, nrhs, dl, d, du, du2, ipiv, b, ldb, info)
 end subroutine gttrs
 
 
+subroutine <prefix>gtcon(norm,n,dl,d,du,du2,ipiv,anorm,rcond,work,iwork,info)
+    ! ?GTCON estimates the reciprocal of the condition number of a real
+    ! tridiagonal matrix A using the LU factorization as computed by
+    ! ?GTTRF.
+    ! An estimate is obtained for norm(inv(A)), and the reciprocal of the
+    ! condition number is computed as RCOND = 1 / (ANORM * norm(inv(A))).
+    threadsafe
+    callstatement (*f2py_func)(norm,&n,dl,d,du,du2,ipiv,&anorm,&rcond,work,iwork,&info)
+    callprotoargument char*, F_INT*, <ctype>*, <ctype>*, <ctype>*, <ctype>*, F_INT*, <ctypereal>*, <ctypereal>*, <ctype>*, <F_INT, F_INT, float,double>*, F_INT*
```

Comment:
`<F_INT, F_INT, float,double>` is copied from the `gecon` wrapper. It is just an integer array. How does this correspond with an integer array?

Can this just be `F_INT*` like `ipiv`?